sp800-63-3/cover.md

```diff
@@ -187,77 +188,78 @@ The terms “SHOULD” and “SHOULD NOT” indicate that among several possibil
 
-This document provides an overview of general authentication frameworks, using authenticators, credentials, and assertions together in a digital system, and a risk-based process of selecting assurance levels. _This document is informative._
+SP 800-63-3 provides an overview of general identity frameworks, using authenticators, credentials, and assertions together in a digital system, and a risk-based process of selecting assurance levels. _This document contains only informative material._
 
 [**SP 800-63A Enrollment and Identity Proofing**](#800-63a)
 
-NIST SP 800-63-A deals with how an individual can prove their identity and become enrolled as a valid user within an identity system. It provides guidelines on processes by which an individual can enrolled and identity proofed in both remote and in-person scenarios. _This document contains both normative and informative material._
+NIST SP 800-63-A addresses how applicants can prove their identities and become enrolled as valid subjects within an identity system. It provides guidelines for processes by which applicants can both proof and enroll at one of three different levels of risk mitigation in both remote and physically-present scenarios. _This document contains both normative and informative material._
 
-SP 800-63A sets requirements to achieve a given IAL. The three IALs reflect the options agencies may select based on their risk profile and the potential harm caused by an attacker successfully claiming the identity of another individual.  The IALs are as follows:
+SP 800-63A sets requirements to achieve a given IAL. The three IALs reflect the options agencies may select based on their risk profile and the potential harm caused by an attacker making a successful false claim of an identity.  The IALs are as follows:
 
-**IAL 1**:
-At this level, there is no requirement for an applicant's identity to be proven.  Any attributes provided in conjunction with the authentication process are self-asserted.
+**IAL 1**: There is no requirement to link the applicant to a specific real-life identity.  Any attributes provided in conjunction with the authentication process are self-asserted or should be treated as self-asserted.
 
-**IAL 2**:
-At IAL 2, the claimed identity is proven with evidence that supports the real-world existence of the claimed identity and identifies and verifies the person to whom the claimed identity belongs.  IAL 2 introduces the need for either remote or in-person identity proofing.  Attributes MAY be asserted by CSPs to RPs in support of pseudonymous identity with verified attributes.
+**IAL 2**: Evidence supports the real-world existence of the claimed identity and verifies that the applicant is appropriately associated with this real-world identity.  IAL 2 introduces the need for either remote or physically-present identity proofing.  Attributes could be asserted by CSPs to RPs in support of pseudonymous identity with verified attributes.
```

Comment:
I'm ok with this.  I think we  may add too much noise if we started to define everything here.  Section 4 gets into the weed.